Dockerfile.cpu

```diff
@@ -1,5 +1,7 @@
 FROM ubuntu:16.04
 
+EXPOSE 8000
```

Comment:
Will these eventually change to `PUBLISH` is the idea that people submit requests via the website which we process privately?